src/status_im/transport/message/v1/group_chat.cljs

```diff
@@ -0,0 +1,183 @@
+(ns status-im.transport.message.v1.group-chat
```

Comment:
Docstring, elaborate group chat what it is and diff forms etc

Same below re public stuff, move comments into docstring

Reply:
In general I'm worried group chat is a hard problem to get right, and it isn't clear exactly what trade-offs exist in this - of which there surely are plenty. It's also not part of MVP product scope as outlined earlier, so not sure how much it makes sense to spend time on it.

Maybe it is enough to just outline what these problems are in this ns, and keep the group chat concept on transport layer similar to functionality of what we have. Then in v2 we can address these (after beta maybe?)

Reply:
I am not sure what you mean, this is already similar functionality to what was done before but using a whisper sym-key instead of the private/public keypair that was carried around the whole codebase before. I will make the doc this week end for this part as well. We can disable group chat if you think it is ok, that would make merging of this PR faster since almost all the bugs are in group chat, though I don't think they are particulary hard to tackle it is mostly about unexpected data layout for contacts in group

Reply:
I agree with @yenda, I absolutely wouldn't retain private/public keypair in codebase (necessary for old group chat logic) as it's security risk.